app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,56 @@
+require 'report_email_sender'
```

Comment:
i added lib files to the autoloader config in environment.rb; was it something else you had in mind?